src/encoded/static/components/donor.js

```diff
@@ -40,7 +40,7 @@ const Donor = (props) => {
                             <dd>{biosample ? <a href={context['@id']}>{context.accession}</a> : context.accession}</dd>
                         </div>
 
-                        {context.aliases.length > 0 ?
+                        {context.aliases > 0 ?
```

Comment:
is `context.aliases` a number?

Reply:
aliases is an array of string
I think ENCODE had it default to an empty list, but we don't do that, so was getting an error when there was no aliases